sources.json

```diff
@@ -647,5 +647,36 @@
     "type": "tooling",
     "tags": ["excel","i18n"],
     "license": "gpl-3.0"
+  },
+  {
+    "owner": "ui5-community",
+    "repo": "generator-ui5-library-webc",
+    "addedToBoUI5": "2023-08-25T00:00:00.007Z",
+    "type": "generator",
+    "tags": ["library","webcomponents","generator"],
+    "license": "apache-2.0"
+  }, {
+    "owner": "SAP",
+    "repo": "ui5-tooling-webc",
+    "addedToBoUI5": "2023-08-25T00:00:00.007Z",
+    "type": "module",
+    "tags": ["tooling","webcomponents","generator"],
+    "license": "apache-2.0"
+  },
+  {
+    "owner": "lemaiwo",
+    "repo": "ui5-space-webcomponent-library",
+    "addedToBoUI5": "2023-08-25T00:00:00.007Z",
+    "type": "application",
```

Comment:
if you want your application to be visible on this site
https://bestofui5.org/#/applications
you need to add this to the website itself, its not coming from the data
https://github.com/ui5-community/bestofui5-website/blob/main/src/view/Applications.view.xml

Reply:
Thanks! Not sure if that's needed as this is rather a sample library and not really an application

Reply:
Sure, justed wanted to point it out if you want to add it